inc/az_ulib_descriptor_api.h

```diff
@@ -150,18 +147,16 @@ typedef struct az_ulib_interface_descriptor_tag
  *                            cannot be `NULL` and shall be allocated in a way that it stays valid
  *                            until the interface is unpublished at some (potentially) unknown time
  *                            in the future.
- * @param[in]   command       The function pointer to #az_ulib_capability_command with the
+ * @param[in]   method        The function pointer to #az_ulib_capability_command with the
```

Comment:
why the rename from command to method?

Reply:
2 reasons:
 1- To avoid issues with the initialization `{ .command = method }`.
 2- Method is the function ivoked by the IPC command. So we can differenctiate this 2 things in the documentation. 

But I'm open for suggestions. 

---
In reply to: [603436594](https://github.com/Azure/azure-ulib-c/pull/65#discussion_r603436594) [](ancestors = 603436594)